spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -97,10 +96,10 @@ def assert_specific_domain_server
   it 'will perform a full refresh on 127.0.0.1 even though the os type is not there yet' do
     # using different cassette that represents the hawkular inventory without the operating system resource type
     # TODO: Make this work with live tests
+    # Perhaps having a separate MW without OS configuration on its config
```

Comment:
Did you make it work? (asking because you removed `:record => :none`)

Reply:
Whoops, didn't make it work for live tests. Only updated the yaml to remove all domain and OS interaction from the server. WIll add back the `:record => :none`.